src/Tools/dotnet-user-secrets/src/Internal/InitCommand.cs

```diff
@@ -120,19 +120,18 @@ public void Execute(CommandContext context)
                 }
 
                 // Add UserSecretsId element
+                propertyGroup.Add("  ");
```

Comment:
I'm not seeing an indentation problem when I run the GA version of the tool. Is this necessary?

Reply:
It's necessary to properly indent the `UserSecretsId` element. That's now necessary because we're respecting whitespace when loading the XML doc.